app/jobs/suitable_for_early_allocation_email_job.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class SuitableForEarlyAllocationEmailJob < ApplicationJob
+  queue_as :mailer
+
+  EQUIP_URL = 'https://equip-portal.rocstac.com/CtrlWebIsapi.dll/?__id=webDiagram.show&map=0%3A9A63E167DE4B400EA07F81A9271E1944&dgm=4F984B45CBC447B1A304B2FFECABB777'
+
+  def perform
+    offenders = EarlyAllocation.suitable_offenders_pre_referral_window
+
+    offenders.each do |offender_no|
+      allocation = Allocation.where(nomis_offender_id: offender_no).first
+      next if allocation.nil?
+
+      prisoner = OffenderService.get_offender(offender_no)
+      if !prisoner.nil? && prisoner.within_early_allocation_window?
+
+        already_emailed = EmailHistory.where(nomis_offender_id: offender_no, event: EmailHistory::SUITABLE_FOR_EARLY_ALLOCATION)
```

Comment:
Not sure what the answer is: but how do we make sure that we don't skip sending the email again when the prisoner re-enters prison under a new sentence?

Right now, there will be an old email in the `EmailHistory`, so we'd skip sending to the POM for newer sentences.

Reply:
Completely agree, I didn't think of this